vsrx/get-vsrx.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+IMAGE=$(ls temp/ffp* >/dev/null 2>&1)
+
+if [ ! -f temp/*.qcow2 ]; then
+
+  echo -e "Creating temp directory\n"
+  mkdir -p temp
+
+  echo -e "Downloading file\n"
+  wget -nc -O temp/virtualbox.box https://atlas.hashicorp.com/juniper/boxes/ffp-12.1X47-D15.4-packetmode/versions/0.5.0/providers/virtualbox.box
```

Comment:
URL seems to have changed to https://app.vagrantup.com/juniper/boxes/ffp-12.1X47-D15.4-packetmode/versions/0.5.0/providers/virtualbox.box in the meanwhile.